[Error] 16/11/2020 20:34:33
[Trace Back] cannot identify image file <_io.BytesIO object at 0x000001AB87687DB0>
[Detail] D:/测试文件\DS_Store.jpg
